model/src/main/java/hu/blackbelt/judo/meta/jsl/util/JslDslModelExtension.xtend

```diff
@@ -89,50 +80,53 @@ class JslDslModelExtension {
-			return (relation as ViewTableDeclaration).rowReferenceType;
-		} if (relation instanceof ViewLinkDeclaration) {
-			return (relation as ViewLinkDeclaration).viewReferenceType;
-		} 
-		
-		return relation.simpleTransferReferenceType;
-	}
-
-	def TransferDeclaration getParameterType(TransferCreateDeclaration create) { 
-		if (create instanceof SimpleTransferCreateDeclaration) {
-			return (create as SimpleTransferCreateDeclaration).transferParameterType;
-		} if (create instanceof ViewCreateDeclaration) {
-			return (create as ViewCreateDeclaration).viewParameterType;
-		}
-		
-        throw new IllegalArgumentException("Create modifier:" + create)
-	}
+//	def SingleType getReferenceType(EntityMemberDeclaration member) { 
+//		if (member instanceof EntityStoredFieldDeclaration) {
+//			if ((member as EntityStoredFieldDeclaration).primitiveReferenceType !== null) {
+//				return (member as EntityStoredFieldDeclaration).primitiveReferenceType;
+//			} if ((member as EntityStoredFieldDeclaration).entityReferenceType !== null) {
+//				return (member as EntityStoredFieldDeclaration).entityReferenceType;
+//			} else {
+//				return (member as EntityStoredFieldDeclaration).singleReferenceType;
+//			}
+//		} else if (member instanceof EntityStoredRelationDeclaration) {
+//			return (member as EntityStoredRelationDeclaration).entityReferenceType;
+//		} else if (member instanceof EntityCalculatedFieldDeclaration) {
+//			return (member as EntityCalculatedFieldDeclaration).primitiveReferenceType;
+//		} else if (member instanceof EntityCalculatedRelationDeclaration) {
+//			return (member as EntityCalculatedRelationDeclaration).entityReferenceType;
+//		}
+//	}
+
+//	def TransferDeclaration getReferenceType(TransferRelationDeclaration relation) { 
+//		if (relation instanceof ActorMenuDeclaration) {
+//			if ((relation as ActorMenuDeclaration).rowReferenceType !== null) {
+//				return (relation as ActorMenuDeclaration).rowReferenceType
+//			}
+//			return (relation as ActorMenuDeclaration).viewReferenceType;
+//		}
+//		 else if (relation instanceof ActorAccessDeclaration) {
+//			return (relation as ActorAccessDeclaration).transferReferenceType;
+//		} else if (relation instanceof ViewTableDeclaration) {
+//			return (relation as ViewTableDeclaration).rowReferenceType;
+//		} if (relation instanceof ViewLinkDeclaration) {
+//			return (relation as ViewLinkDeclaration).viewReferenceType;
+//		} 
+//		
+//		return relation.simpleTransferReferenceType;
+//
+//		return relation.referenceType;
+//	}
+
+//	def TransferDeclaration getParameterType(TransferCreateDeclaration create) { 
+//		if (create instanceof TransferCreateDeclaration) {
+//			return (create as TransferCreateDeclaration).parameterType;
+//		} if (create instanceof ViewCreateDeclaration) {
+//			return (create as ViewCreateDeclaration).viewParameterType;
+//		}
+//		
+//        throw new IllegalArgumentException("Create modifier:" + create)
+//	}
 
```

Comment:
Why it is commented out? Is there any purpose to left. If yes, please create a ticket for it!

Reply:
commented lines deleted